rfcs/text/0001-rfc-process.md

```diff
@@ -70,26 +86,66 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
 ## Detailed Design
 
+**ADDITION BEGINS**
+
+### Drafting RFCs
+
+#### **Authors**:
+
+There are two types of RFCs that may be proposed: ***informational*** RFCs and ***non informational*** RFCs. Non informational RFCs are expected to be linked directly to an existing ticket in the DCP Zenhub board which will usually be tagged as a Spike. Informational RFCs are more speculative and need not directly reference an open ticket in the DCP Zenhub board.
```

Comment:
A governance non-informational RFC like this one is not expected to be linked to a spike in ZenHub.

I believe that I commented on _speculative_ in earlier remarks.

Reply:
Also - Editorial consistency tagged -> labeled

Reply:
I wonder if we could simplify by referring to three types of RFC(s):
- Software
- Informational
- Governance


where software RFCs are intended to be represented by a Spike in ZenHub to demonstrate that it is a DCP priority. The three types would be similar to the Python PEP approach (Standards, Informational, Process).